clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -554,6 +571,7 @@ private void doSend(ClientRequest clientRequest, boolean isInternalRequest, long
         handleDisconnections(responses, updatedNow);
         handleConnections();
         handleInitiateApiVersionRequests(updatedNow);
+        handleTimeOutConnections(responses, updatedNow);
```

Comment:
`TimeOut` => `TimedOut`

Reply:
Do you mean TimeOut => Timeout? Refactored.